modules/social_features/social_private_message/src/Service/SocialPrivateMessageService.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\social_private_message\Service;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Session\AccountProxyInterface;
+use Drupal\private_message\Entity\PrivateMessageThread;
+use Drupal\private_message\Service\PrivateMessageService;
+use Drupal\user\Entity\User;
+use Drupal\user\UserDataInterface;
+use Drupal\user\UserInterface;
+
+/**
+ * Class SocialPrivateMessageService.
+ *
+ * @package Drupal\social_private_message\Service
+ */
+class SocialPrivateMessageService extends PrivateMessageService {
+
+  /**
+   * The Database.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected $database;
+
```

Comment:
@jochemvn One enter too much.

Maybe you can also run the coding standards on just this feature and fix all open issues.